src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -304,7 +303,7 @@ public void should_succeed_and_return_204_when_a_task_that_was_already_claimed_a
     }
 
     @Test
-    public void should_fail_and_return_401_when_a_task_was_already_claimed_and_privileged_auto_complete_is_false() {
```

Comment:
do we still need an edge case IT for when RoleAssignment cannot be retrieved and then return 401? As per first validation scenario on confluence?

Reply:
this is the same test just corrected the title to specify it should return a 403 instead of a 401